doc/classes/EditorImportPlugin.xml

```diff
@@ -1,8 +1,53 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorImportPlugin" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Used by the editor to import resources from files.
 	</brief_description>
 	<description>
+		EditorImportPlugins provide a way to extend the editor's resource import functionality. They can be used to import resources from custom files or to provide alternatives to the editor's existing importers. EditorImportPlugins must be registered by an [EditorPlugin] by using [method EditorPlugin.add_import_plugin].
+
+		EditorImportPlugins work by associating with specific file extensions (see: [method get_recognized_extension] and godot resource types (see: [method get_resource_type]). They may optionally specify some import presets that affect the import process. EditorImportPlugins are responsible for creating the resources and saving them in the [code].import[/code] directory.
```

Comment:
Avoid parens: use short sentences instead. Exceptional case where you may use passive voice, as we can consider that EditorImportPlugin is linked to other resources by its parent class, the system, the editor...
`An EditorImportPlugin is linked to specific file extensions and resource type. See: [method get_recognized_extension] and [method get_resource_type].`